packages/js-runtime/deno.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "@commontools/js-runtime",
   "tasks": {
-    "test": "deno test --allow-read --allow-write --allow-run --allow-env=API_URL,\"TSC_*\",NODE_INSPECTOR_IPC,VSCODE_INSPECTOR_OPTIONS,NODE_ENV test/*.test.ts"
+    "test": "deno test --allow-read --allow-write --allow-run --allow-env=UPDATE_GOLDENS,API_URL,\"TSC_*\",NODE_INSPECTOR_IPC,VSCODE_INSPECTOR_OPTIONS,NODE_ENV test/*.test.ts"
   },
   "imports": {
     "typescript": "npm:typescript",
-    "source-map-js": "npm:source-map-js@^1.2.1"
+    "source-map-js": "npm:source-map-js@^1.2.1",
+    "@commontools/test-support": "../test-support/src/mod.ts",
+    "@commontools/test-support/fixture-runner": "../test-support/src/fixture-runner.ts",
```

Comment:
The `test-support` package can provide this as an `export` -- see the `utils` package multi exports for an example